queries.py

```diff
@@ -87,3 +88,52 @@ def get_expenditure_by_country_func_econ_year():
     """
     df = execute_query(query)
     return df
+
+
+def get_basic_country_data(countries):
+    country_list = "', '".join(countries)
+    query = """
+        SELECT * FROM indicator.country
+    """
+    query += f" WHERE country_name IN ('{country_list}')"
+    df = execute_query(query)
+    return df
+
+
+def get_expenditure_by_country_geo1_year():
+    query = """
+        SELECT * FROM boost.expenditure_by_country_geo1_year
+    """
+    df = execute_query(query)
+    return df
+
+
+def get_adm_boundaries(countries):
+    query = """
+        SELECT * FROM indicator.admin1_boundaries_gold
```

Comment:
same as above

Reply:
I have added the required column names explicitly in the queries. I think there was not too much data being fetched which was not being used. So, there is not too much improvement in the loading times etc.